src/InputManager.cpp

```diff
@@ -12,10 +12,19 @@ void InputManager::CheckInput()
 	MouseInput();
 }
 
+#include <iostream>
 void InputManager::MouseInput()
 {
 	double xpos, ypos;
-	if(glfwGetMouseButton(window, GLFW_MOUSE_BUTTON_LEFT) == GLFW_PRESS)
+	if( shiftKey && glfwGetMouseButton(window, GLFW_MOUSE_BUTTON_LEFT) == GLFW_PRESS){
```

Comment:
Left bracket needs to go to next line to keep consistent with the rest of the code.

Reply:
d